src/regions/ca/info/ca-ns/clinics.md

```diff
@@ -16,4 +16,6 @@ If you answer yes to two or more of these symptoms, please call 811 for further
 
 If you experience other symptoms that are a concern for you, call 811 to speak with a nurse.
 
+Signs of a fever include chills, sweats, muscle aches, lightheadedness.
```

Comment:
Maybe add the temperature/degrees that qualify being a fever?

Reply:
It is already up there where it says 38 degrees C, I believe.